### PR TITLE
Create muro.md

### PR DESCRIPTION
**Pull request by paulzrq**
update muro.md

### CODE REVIEW

File: docs/maths/muro.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Good
```
Comment (Yuchen-Wang-SH) [inline docs/maths/muro.md:1]: What is this?

File: docs/maths/muro.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+## good
+**good**
+
+*good*
```
Comment (Yuchen-Wang-SH) [inline docs/maths/muro.md:5]: What is that?

**Review by Yuchen-Wang-SH — CHANGES_REQUESTED** [review]

File: docs/maths/muro.md (hunk 1 above)
Comment (paulzrq) [reply docs/maths/muro.md:1]: it is a file

Comment (Yuchen-Wang-SH) [reply docs/maths/muro.md:1]: Make sense

**Review by Yuchen-Wang-SH — APPROVED** [review]

**Pull request merged**